### PR TITLE
Pull request for simplified Chinese version docker code-of-conduct

### PR DESCRIPTION
**Pull request by 10000TB**
Hi,
    I added a simplified Chinese version of Docker code-of-conduct into this repo, let me know how it looks, and feel free to leave any comment.

Thanks
Xuehao, (AKA: David)

### CODE REVIEW

File: code-of-conduct-ZH-Simple.md [hunk 1]
```diff
@@ -0,0 +1,40 @@
+## 引言
+
+我们致力于为每个人提供一个没有扰乱，包容的体验，我们绝不容忍参与者对社区得任何形式的扰乱。我们要求参与者相互体谅，举止专业，并且对他人充满尊重。这套行为准则及其相关条例同样适用于开源社区活动范围以外的令人无法接受的行为，这包括所有的社区场所：在线，面对面，以及任何其他形式的点对点的交流，以及任何地点以至于相关的行为可以对社区成员的安全和个人健康造成伤害。Docker, Inc（Docker峰会，线下见面，用户群组 ）组织的或在Docker, Inc 场所所举办的活动的参展者，发言者，赞助者,工作人员，以及所有其他的参与者同样受到社区指南和行为准则的制约。
```
Comment (tifayuki) [inline code-of-conduct-ZH-Simple.md]: harassment­ is better to be translated to "骚扰" here, IMHO

File: code-of-conduct-ZH-Simple.md [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+
+多元化和包容精神是的Docker社区更加强大和繁荣。我们鼓励来自尽可能不同的，多元的背景的人士的参与, 并且我们希望我们的立场对于每个人都非常鲜明。
```
Comment (tifayuki) [inline code-of-conduct-ZH-Simple.md]: typo: 使得

**Comment by tifayuki** [comment]
LGTM on comments

File: code-of-conduct-ZH-Simple.md (hunk 1 above)
Comment (10000TB) [inline code-of-conduct-ZH-Simple.md]: Hi @tifayuki , I hesitated at the beginning for the translation of harassment here, but IMHO, here is in intro section, we want a more general expression, and '扰乱' contains '骚扰' ('骚扰' is type of  '扰乱'), so I would suggest keeping  '扰乱'.

File: code-of-conduct-ZH-Simple.md (hunk 2 above)
Comment (10000TB) [inline code-of-conduct-ZH-Simple.md]: fixed

**Comment by 10000TB** [comment]
Is there anymore review needed? or can this request be merged?

Thanks,
David

**Pull request merged**